server/app.js

```diff
@@ -19,7 +18,7 @@ app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cookieParser());
```

Comment:
Let's remove all `ejs` related imports since we are not rendering using template engine anymore (eg. line 12, 13).